src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceCallbackHandler.java

```diff
@@ -285,8 +285,7 @@ private CallbackResponse aboutToSubmit(CallbackParams callbackParams, boolean v1
         AllocatedTrack allocatedTrack =
             getAllocatedTrack(caseData.getClaimValue().toPounds(), caseData.getClaimType());
 
-        if (v1 && featureToggleService.isSdoEnabled()
-            && !AllocatedTrack.MULTI_CLAIM.equals(allocatedTrack)) {
+        if (v1 && !AllocatedTrack.MULTI_CLAIM.equals(allocatedTrack)) {
```

Comment:
what callback is currently used for SDO?
Are we able to free up the V1 callback - this could perhaps be moved into the base callback handler

Reply:
The V1 callback has been removed in CIV-7975 which is currently in test, I left it out of this PR to avoid duplication